.github/workflows/publish.yml

```diff
@@ -0,0 +1,55 @@
+name: publish
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
+    steps:
+      - 
+        uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+      - 
+        name: Get tags
+        id: get_tag
+        run: echo ::set-output name=TAG::${GITHUB_REF#refs/tags/}
+      -
+        name: Set Username/Repo and ImagePrefix as ENV vars
+        run: |
+          echo "USER_REPO"=$(echo "$GITHUB_REPOSITORY" | awk '{print tolower($1)}' | sed -e "s/:refs//") >> $GITHUB_ENV && \
+          echo "IMAGE_PREFIX"=$(echo "ghcr.io/${{ github.repository_owner }}/queue-worker" | awk '{print tolower($1)}' | sed -e "s/:refs//") >> $GITHUB_ENV
+      - 
+        name: Login to Github Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - 
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - 
+        name: Build and push multi-arch containers images
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./Dockerfile
+          load: false
+          push: true
+          platforms: linux/amd64,linux/arm/v7,linux/arm64
+          labels: |
+            org.opencontainers.image.source=https://github.com/${{ env.USER_REPO }}
+          build-args: |
+            VERSION=latest-dev
```

Comment:
I don't think that's the `VERSION` that we want for a publish build.

Did you copy this from somewhere, because we should fix that project too, if you did.

Reply:
No it's my fault. I copied the steps from the build file to publish, that's why this happened, missed it. Thanks!